### PR TITLE
Fix advanced AngularJS array and props tests

### PR DESCRIPTION
**Pull request by antgonzales**
The current tests don't setup the components to test these attributes against expectations.

### CODE REVIEW

**Comment by antgonzales** [comment]
@robdodson any chance I can get feedback on this pull request?

File: libraries/angularjs/src/advanced-tests.js [hunk 1]
```diff
@@ -17,15 +17,25 @@ describe("advanced support", () => {
...
+    }
+
     it("will pass array data as a property", function() {
       this.weight = 2;
-      let data = wc.arr;
-      expect(data).toEqual(["A", "n", "g", "u", "l", "a", "r"]);
+      let root = prep("<comp-with-props>")
+      scope.$digest()
+      let wc = root.querySelector('#wc')
+      let data = wc.arr || wc.getAttribute('arr')
```
Comment (robdodson) [inline libraries/angularjs/src/advanced-tests.js]: It shouldn't be calling `getAttribute` here. For arrays and objects, the test is trying to assert that they are always passed as JS properties.

File: libraries/angularjs/src/advanced-tests.js [hunk 2]
```diff
@@ -17,15 +17,25 @@ describe("advanced support", () => {
...
+      expect(data).toEqual(['A', 'n', 'g', 'u', 'l', 'a', 'r']);
     });
 
     it("will pass object data as a property", function() {
       this.weight = 2;
-      let data = wc.obj;
+      let root = prep("<comp-with-props>")
+      scope.$digest()
+      let wc = root.querySelector('#wc')
+      let data = wc.obj || wc.getAttribute('obj')
```
Comment (robdodson) [inline libraries/angularjs/src/advanced-tests.js]: Same as above.

File: libraries/angularjs/src/advanced-tests.js (hunk 1 above)
Comment (antgonzales) [reply libraries/angularjs/src/advanced-tests.js]: 👍

**Comment by antgonzales** [comment]
@robdodson thanks for the feedback! Fixed.

**Comment by robdodson** [comment]
Looks good. Since I'm totally not an AngularJS expert, can you explain a bit about what this change actually does? I noticed the test still fails (which is what I expected because AngularJS isn't setting properties here). But I was just curious what all the setup work does.

**Comment by robdodson** [comment]
Cool, thanks for the PR!

**Pull request merged**